### PR TITLE
fixed 37158: A11Y/LoginPage: The A11Y info page cannot be requested o…

### PR DESCRIPTION
**Pull request by alex40724**
…n the login page

See
https://mantis.ilias.de/view.php?id=37158

### CODE REVIEW

File: Services/Init/classes/class.ilStartUpGUI.php [hunk 1]
```diff
@@ -155,6 +154,9 @@ public function executeCommand()
                 require_once("Services/Init/classes/class.ilPasswordAssistanceGUI.php");
                 return $this->ctrl->forwardCommand(new ilPasswordAssistanceGUI());
 
+            case "ilaccessibilitycontrolconceptgui":
```
Comment (mjansenDatabay) [inline Services/Init/classes/class.ilStartUpGUI.php]:
```suggestion
            case strtolower(ilAccessibilityControlConceptGUI::class):
```

File: Services/Init/classes/class.ilInitialisation.php [hunk 2]
```diff
@@ -1720,6 +1720,13 @@ protected static function blockedAuthentication(string $a_current_script): bool
             return true;
         }
 
+        if (
+            (strtolower($requestCmdClass ?? "") === "ilaccessibilitycontrolconceptgui")
```
Comment (mjansenDatabay) [inline Services/Init/classes/class.ilInitialisation.php]:
```suggestion
            (strtolower($requestCmdClass ?? "") === strtolower(ilAccessibilityControlConceptGUI::class))
```

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]
Hi @alex40724 ,

thx for the PR.

I suggest to use class-strings to avoid problems when the classes are renamed in future. If classes are renamed (e.g. in PhpStorm) such class-strings are automatically renamed as well. This is maybe more "robust" than using "Search & Replace".

Best regards,
Michael

**Comment by alex40724** [comment]
@mjansenDatabay Yes, of course. I added your changes.

**Review by mjansenDatabay — APPROVED** [review]

**Comment by mjansenDatabay** [comment]
@pascalseeland I'll merge and pick this to all relevant branches.

**Comment by alex40724** [comment]
@mjansenDatabay Thanks!

**Pull request merged**